editions/tw5.com/tiddlers/wikitext/Widgets in WikiText.tid

```diff
@@ -23,8 +23,4 @@ Note that widgets inherit all the features of [[HTML in WikiText]]:
 * The content of a widget is [[parsed|WikiText Parser Modes]] in [[inline mode|Inline Mode WikiText]] unless the opening tag is followed by two linebreaks, which forces [[block mode|Block Mode WikiText]]
 ** [[inline mode|Inline Mode WikiText]] means that [[block mode|Block Mode WikiText]] parse rules like headings, tables and lists are not recognised
 
-See [[HTML in WikiText]] for more details.
-
-The available widgets include:
-
-<<list-links "[tag[Widgets]]">>
+A list of all available widgets can be found at: [[Widgets]]
```

Comment:
Why remove the widget listing here? Again it seems like a documentation change that should be presented separately in order to be considered properly, not shoehorned into a completely separate PR.